lib/handler.js

```diff
@@ -1,56 +1,42 @@
 'use strict';
 
-const loggerExtension = require('./logger-extension');
-const utils = require('./utils');
-
-module.exports = function(handler, options) {
-	if (typeof handler !== 'function') {
-		throw new Error('handler is not a function');
-	}
-
-	options = options || {}; // eslint-disable-line no-param-reassign
-
-	['onBefore', 'onAfter', 'onError'].forEach(hookMethod => {
-		if (options[hookMethod] && typeof options[hookMethod] !== 'function') {
-			throw new Error(`options.${hookMethod} must be a function when present`);
-		}
-	});
-
-	if (options.errorStack && typeof options.errorStack !== 'boolean') {
-		throw new Error('options.errorStack must be a boolean when present');
+module.exports = function () {
+	const middleware = [];
+
+	// modified from github.com/koajs/compose
+	function dispatch (state, i = 0) {
```

Comment:
I don't think so, but it's so trivial to change, I just made it passed in explicitly.